android/src/main/kotlin/com/icapps/background_location_tracker/service/LocationUpdatesService.kt

```diff
@@ -153,6 +163,8 @@ internal class LocationUpdatesService : Service() {
         } else {
             startService(Intent(applicationContext, LocationUpdatesService::class.java))
         }
+        val locationRequest = this.locationRequest ?: return
```

Comment:
I don't think this is required here

Reply:
requestLocationUpdates requires a non nullable value